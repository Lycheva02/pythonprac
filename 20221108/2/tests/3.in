(0,0), (1,2), (4,8)
(8,8), (9,13), (16,244)

