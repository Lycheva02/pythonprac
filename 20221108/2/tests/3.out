Not a triangle
98.00
